static/src/stylesheets/module/nav/_control.scss

```diff
@@ -40,6 +40,10 @@ $control-size: 36px;
     }
 }
 
+.control__info--supporting {
```

Comment:
Always felt weird that there's a `.control__info` but no .`control` class in these styles… You're following the rest of the file though, so it's fine.